src/Tempest/Framework/Application/Kernel.php

```diff
@@ -35,6 +37,8 @@ public function init(): Container
             )->boot();
         }
 
+        $container->singleton(DatabaseDriver::class, $container->get(DatabaseConfig::class)->driver);
```

Comment:
binding the current driver to the driver interface/contract. Currently figuring out if I can make this work for the `QueryStatement` class

Reply:
You should move this logic outside the kernel and create an initializer class for `DatabaseDriver` 👍